src/en/articles/watch-october-november-2025.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "#a11y technical watch, October-November 2025"
+abstract: "#a11y anthology, October-November 2025"
+date: "2025-12-10"
+tags:
+  - veille
+---
+
+# Digital accessibility watch October-November 2025
+
+To start, here are two great tech and a11y advent calendars: (fr) https://www.24joursdeweb.fr/ and https://www.htmhell.dev/adventcalendar/.
+
+And then there's progress (it's becoming clearer...) regarding the complaints filed by ApiDV (Accompanying, Promoting, Integrating Visually Impaired People) against major retailers for non-compliance with the EAA (Accessibility Advisory Agreement), and finally, its transposition into French law (fr):
+
+- https://les-enovateurs.com/breves/auchan-carrefour-leclerc-picard-inaccessibilite-sites-justice
+- https://www.interetaagir.org/assignation-en-refere-des-entreprises-auchan-carrefour-e-leclerc-et-picard-surgeles/.
+
+That's all, happy holidays everyone!
+
+## Accessibility in Laws, Standards, and Recommendations
+
+- RaWeb integrates EN301549 (fr): https://access42.net/references-clients/raweb-referentiel-accessibilite-web-luxembourg/
+- Meeting EAA requirements: https://www.deque.com/blog/understanding-compliance-and-conformance-in-the-eaa-era/
+- Explanation of WCAG 2.4.5 Multiple ways: https://tarnoff.info/2025/10/27/a11y-101-2-4-5-multiple-ways/
+
+## Accessibility Implementation Strategy
+
+- Alternative text should be as long as necessary: https://chrisyoong.com/blog/the-100-150-or-200-characters-alt-text-rule-is-a-myth
+- No headings in the header: https://css-tricks.com/headings-semantics-fluidity-and-styling-oh-my/
+- Automated tests are lying to you: https://chrisyoong.com/blog/your-accessibility-score-is-lying-to-you
+- Sustainable A11y for large organizations, three key articles:
+  - https://tetralogical.com/blog/2025/11/07/sustainable-accessibility-in-complex-organisations-organisational-realities/
+  - https://tetralogical.com/blog/2025/11/07/sustainable-accessibility-in-complex-organisations-strategic-foundations/
+  - https://tetralogical.com/blog/2025/11/07/sustainable-accessibility-in-complex-organisations-external-factors/
+- Shopify and a11y: https://adrianroselli.com/2025/11/shopify-needs-a-mirrorfy.html
+- a11y bugs shouldn't just be put in the backlog: https://intopia.digital/articles/straight-to-the-pool-room-the-pitfalls-of-placing-accessibility-issues-in-the-backlog/
+- Getting things moving again when a11y stalls: https://buttondown.com/access-ability/archive/when-accessibility-progress-plateaus-how-to/
+
+## Testing Accessibility
+
+- User Testing with People with Disabilities: https://intopia.digital/articles/user-testing-with-people-with-disabilities/w
+- Testing WCAG by Dennis Deacon, full of articles on almost all criteria, worth reading: https://www.dennisdeacon.com/web/accessibility/
```

Comment:
Looks like this URL is a 404?
Correct URL seems to be https://www.dennisdeacon.com/category/web/accessibility/